packages/contracts/.env.template

```diff
@@ -1,7 +1,13 @@
-ANVIL_CHAIN_ID=31337
+# ANVIL: Base url for anvil. Create a local testnet node for deploying. It can also be used to fork other EVM compatible networks.
 ANVIL_FORK_URL=http://localhost:8545
+ANVIL_CHAIN_ID=31337
+
+
+# ETHERSCAN_API_KEY: This API key is used to verify and publish smart contracts on the Etherscan blockchain explorer. To get an API key, follow these steps in the etherscan documentation -> https://docs.etherscan.io/getting-started/viewing-api-usage-statistics
 ETHERSCAN_API_KEY=
-FORGE_RPC_URL=
-FORGE_PRIVATE_KEY=
+
+# PRIVATE_KEY: This is the private key for the Ethereum account that will be used to deploy the smart contract to the Goerli testnet. To obtain a private key, you can follow the steps in the metamask documentation -> https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 PRIVATE_KEY=
+
+# GOERLI_RPC_URL: This is the RPC endpoint for the Goerli testnet. To obtain the RPC URL, follow these steps: Go to your wallet, click on the Goerli network and look for the RPC URL value.
 GOERLI_RPC_URL=
```

Comment:
`#would` also add what these things are for -- [anvil](https://book.getfoundry.sh/anvil/) for the local testnet,  etherscan for verifying contracts, private key to sign deploys, and RPC to connect to the testnet.